app/models/order.rb

```diff
@@ -1,4 +1,4 @@
 class Order < ApplicationRecord
   belongs_to :user
-  belongs_to :yarn
```

Comment:
Question: Would the association not be an Order "has_many :products"?

Reply:
I agree!

Reply:
Okay!